### ISSUE TITLE
arm64 FreeBSD & NetBSD

### ISSUE THREAD

**Issue by chipsenkbeil** [cross-platform-actions/action#55]
I noticed that FreeBSD only offers x86_64 and not arm64 compared to OpenBSD. Is there any chance of [FreeBSD arm64](https://www.freebsd.org/platforms/arm/) being available? For that matter, [NetBSD aarch64](https://wiki.netbsd.org/ports/aarch64/)?

**Comment by jacob-carlborg** [comment]
Yes, I plan to add support for FreeBSD and NetBSD on ARM64. It was easier with OpenBSD. With FreeBSD, last time I tried, there was some weird issue in the interaction between Packer and the firmware. I don't remember the exact details of what didn't work with NetBSD.

**Comment by chipsenkbeil** [comment]
Great to hear! Would be a huge help for me - and I'm assuming many others - to be able to more easily build on those platforms in a consistent manner.

For me in particular, my project is a Rust-based one, and the tooling to manage versions of the Rust compiler isn't available on many of the non-x86 architectures, but the compiler itself is. So being able to spin up an arm64 version of FreeBSD or NetBSD to build against would both simplify my workstream and let me test the builds versus just cross-compiling.

**Comment by jacob-carlborg** [comment]
FreeBSD for ARM64 is now available: https://github.com/cross-platform-actions/action/releases/tag/v0.16.0.

**Comment by schmonz** [comment]
If my memory serves, NetBSD/aarch64 now runs much better virtualized than it did in 9.3. 10.0 is nearing release. Might be a good time to try again with the most recent release candidate.

**Comment by jacob-carlborg** [comment]
@schmonz 
> NetBSD/aarch64 now runs much better virtualized than it did in 9.3

I'm already working on it. I have NetBSD 10 RC2 more or less working. I've managed to get it to boot in the GitHub action yesterday [1]. I did run into some issue when installing NetBSD, it failed to setup `pkgin`. I think there were some missing package servers. I hope that's because it's not a final release yet.

> Dunno whether GItHub provides aarch64 hosts

They do have macOS large runners running on Apple Silicon in beta now. But I don't have access to those.

[1] https://github.com/cross-platform-actions/action/actions/runs/7679908323/job/20931321519
